app/views/schools/advice/solar_pv/_analysis_no_solar_table.html.erb

```diff
@@ -50,6 +53,7 @@
         <td class="text-right"><%= format_unit(scenario.solar_pv_output_co2, :co2) %></td>
         <td class="text-right"><%= format_unit(scenario.capital_cost_£, :£) %></td>
         <td class="text-right"><%= scenario.payback_years.round %> years</td>
+        <td class="text-right"><%= format_unit(scenario.export_income_£, :£) %></td>
```

Comment:
fyi @ldodds Export income is missing from this table (so no changes shown in the table with any changes in the site settings solar_export_price.  let me know if this needs moving/removing
![Screenshot 2023-05-11 at 09 18 02](https://github.com/Energy-Sparks/energy-sparks/assets/25906/110dd01e-f312-41d8-b4b2-635606da41aa)